packages/consumption/src/modules/requests/itemProcessors/readAttribute/ReadAttributeRequestItemProcessor.ts

```diff
@@ -138,6 +138,33 @@ export class ReadAttributeRequestItemProcessor extends GenericRequestItemProcess
                         )
                     );
                 }
+
+                const queryForPendingRelationship = {
+                    "peer.address": foundLocalAttribute.shareInfo.peer.address,
+                    status: RelationshipStatus.Pending
+                };
+                const pendingRelationshipToPeer = await this.accountController.relationships.getRelationships(queryForPendingRelationship);
+
+                const queryForActiveOrTerminatedRelationship = {
+                    "peer.address": foundLocalAttribute.shareInfo.peer.address,
+                    $or: [
+                        {
+                            ["status"]: { $eq: RelationshipStatus.Active }
+                        },
+                        {
+                            ["status"]: { $eq: RelationshipStatus.Terminated }
+                        },
+                        {
+                            ["status"]: { $eq: RelationshipStatus.DeletionProposed }
+                        }
+                    ]
+                };
+
+                const activeOrTerminatedRelationshipToPeer = await this.accountController.relationships.getRelationships(queryForActiveOrTerminatedRelationship);
```

Comment:
```suggestion
                const activeOrTerminatedRelationshipsToPeer = await this.accountController.relationships.getRelationships(queryForActiveOrTerminatedRelationship);
```

Reply:
It can be assumed that there is either no `Active` or `Terminated` or `DeletionProposed` Relationship, or at most one. I have therefore decided to use the singular instead of the plural, as the length of the array is at most one if the program runs without bugs. If you like, I can still use the plural form. Or do you have any better ideas for naming this variable that would be more suitable?

Reply:
true, but a variable called activeOrTerminatedRelationshipToPeer should not be an array.